### PR TITLE
Hacktoberfest bug fixes

### PR DESCRIPTION
**Pull request by Simplicitee**
## Fixes
* Fixes every issue except for the readme enhancement

### CODE REVIEW

File: src/main/java/com/strangeone101/bendinggui/menus/MenuEditElements.java [hunk 1]
```diff
@@ -113,22 +113,28 @@ public void onClick(Player playerwhoclicked)
 				{
 					if (playerwhoclicked.hasPermission("bending.admin.add"))
 					{
-						if (player instanceof Player)
-							((Player)player).sendMessage(ChatColor.YELLOW + "You are now " + (element == Element.AIR || element == Element.EARTH ? "an " : "a ") + c + element.getName().toLowerCase() + ChatColor.YELLOW + " " + element.getType().getBender() + "!");
+						if (player.isOnline())
```
Comment (StrangeOne101) [inline src/main/java/com/strangeone101/bendinggui/menus/MenuEditElements.java]: Why did you change this? This actually makes the cast bellow technically unsafe.

**Review by StrangeOne101 — COMMENTED** [review]
Apart from the one thing, looks good. Did you test on 1.8.7?

**Comment by Simplicitee** [comment]
Corrections have been made, test was done on PK 1.8.7 and spigot 1.12.2
